code/modules/surgery/surgery.dm

```diff
@@ -181,14 +181,48 @@
 	return null
 
 
-/proc/spread_germs_to_organ(obj/item/organ/E, mob/living/carbon/human/user)
-	if(!istype(user) || !istype(E)) return
-//	to_chat(world, "Germ spread: [E] : [E.owner]")
+/proc/spread_germs_to_organ(obj/item/organ/E, mob/living/carbon/human/user, obj/item/tool)
+	if(!istype(user) || !istype(E))
+		return
+
 	var/germ_level = user.germ_level
+
+	//germ spread from surgeon touching the patient
 	if(user.gloves)
 		germ_level = user.gloves.germ_level
 	if(!(E.status & ORGAN_ROBOT)) //Germs on robotic limbs bad
 		E.germ_level = max(germ_level,E.germ_level) //as funny as scrubbing microbes out with clean gloves is - no.
+		spread_germs_by_incision(E,tool)//germ spread from environement to patient
+
+/proc/spread_germs_by_incision(obj/item/organ/external/E,obj/item/tool)
+	if(!istype(E,/obj/item/organ/external))
+		return
+
+	var/germs = 0
+
+	for(var/mob/living/carbon/human/H in view(4, E.loc))//germs from people
+		if(AStar(E.loc, H.loc, /turf/proc/Distance, 4, simulated_only = 0))
+			if((!(NO_BREATH in H.mutations) || !(H.species.flags & NO_BREATH)) && !H.wear_mask) //wearing a mask helps preventing people from breathing cooties into open incisions
+				germs+=H.germ_level/4
+
+	for(var/obj/effect/decal/cleanable/M in view(4, E.loc))//germs from messes
+		if(AStar(E.loc, M.loc, /turf/proc/Distance, 4, simulated_only = 0))
+			if(istype(M,/obj/effect/decal/cleanable/dirt))//dirt is too common
```

Comment:
I still posit this is going to cause more frustration than its worth; it's very difficult to see low level dirt in an area, yet it's still there; short of spraying down the entire room or having a cleanbot prior to every surgery, it's going to cause a lot of infections out of the blue--which is going to be incredibly frustrating to players.